changelogs/unreleased/gh-6200-lua_cjson-encode-escapes-slash.md

```diff
@@ -1,5 +1,6 @@
 ## bugfix/lua/json
 
-* Now the forward slash (`/`) is not escaped in `json.encode()` and msgpack.
-  A new `tarantool.compat` option `json_escape_forward_slash` is added for
-  switching to the new behavior (gh-6200).
+* A new `tarantool.compat` option `json_escape_forward_slash` was added. This
+  option configures whether the internal JSON encoder escapes the forward slash
+  character (old behavior) or not (new behavior). This option affects the
+  `json.encode()` Lua function and the JSON logger (gh-6200).
```

Comment:
> Part of #8117

Nit: Shouldn't we close the issue here? I understand that it'll be closed when the pull request will land, but usually we close an issue at the last commit in a series. So one may understand when the problem is considered entirely solved by looking over `git log`.

Reply:
Makes sense. Replaced `Part of #XXXX` with `Closes #XXXX` statements where appropriate.